package.json

```diff
@@ -54,7 +58,6 @@
     "react-syntax-highlighter": "^15.6.1",
     "remark-gfm": "^3.0.1",
     "sass": "^1.81.0",
-    "tailwind": "^4.0.0",
```

Comment:
this is some shadowy package that was likely installed by mistake. [tailwindcss](npmjs.com/package/tailwindcss) is the right package (and it is still installed)